ASSUMPTIONS.md

```diff
@@ -99,3 +102,87 @@ As more statistical data is collected about slashing in different validator envi
+
+Pre EIP-1559 the fee cap is equivalent to the gas price in Gwei per gas. We use the historical mean gas price over the past 12 months to set the average fee cap, which in this model is equivalent to the sum of the values from the `base_fee_process` and `priority_fee_process`.
+
+We use a static value of 100 Gwei per gas (historical mean gas price) to set the default values for the `base_fee_process` and `priority_fee_process`. See [Miner Extractable Value as Percentage of Average Fee Cap](#miner-extractable-value-as-percentage-of-average-fee-cap) for further related assumptions.
+
+### Miner Extractable Value as Percentage of Average Fee Cap
+
+TODO: get Roger to review and validate
+
+Miner Extractable Value (MEV) is a measure of the profit a miner/validator can make by arbitrarily including, excluding, or re-ordering transactions within the blocks they produce.
+
+Under the influence of MEV, it is assumed a certain proportion of miners/validators would run MEV nodes (e.g. Flashbots), resulting in a blockspace auction where users would need to increase their transaction priority fee in order to get transactions included in the blockspace. In steady state, without the influence of MEV, it is assumed the priority fee would be on average 1 Gwei per gas to compensate for uncle risk - see https://notes.ethereum.org/@vbuterin/BkSQmQTS8#Why-would-miners-include-transactions-at-all.
+
+Calculating MEV as a percentage of transaction fees is nontrivial, but based on the following two pieces of research, we assume that 30% of transaction fees can be attributed to MEV:
+* A [Twitter post by Robert Miller](https://mobile.twitter.com/bertcmiller/status/1405234475680862210) presents a time-series of MEV as a percentage of transaction fees over the period 1 March 2021 to 1 June 2021 (Flashbots started around December 2020, and got popular first quarter 2021) ranging between 11% and 46% with an upwards trend. This time-series is based on data collected using the [Flashbots MEV Inspect library](https://github.com/flashbots/mev-inspect-py). It's worth noting the caveats mentioned in the Twitter thread for more rigorous analysis.
+* Just Drake's spreadsheet ["staking APR with EVM fee rewards"](https://docs.google.com/spreadsheets/d/1FslqTnECKvi7_l4x6lbyRhNtzW9f6CVEzwDf04zprfA) presents a "Best guess" of a 70% fee burn percentage - this means 70% of the average fee cap is attributed to the base fee.
+
+We assume the effect of MEV will continue, and thus assume a static value of 0.3 (30%) multiplied by the [average fee cap](#average-fee-cap-max_fee_per_gas) to set the default value for the EIP-1559 priority fee in Gwei per gas - to change this value, update the `priority_fee_process` [System Parameter](./model/system_parameters.py). The `priority_fee_process` [System Parameter](./model/system_parameters.py) can also be a time-series of priority fee values, if you would like to test more complex scenarios.
```

Comment:
Priority fee is related to EIP-1559, and is a separate mechanism to MeV rewards. Prior to the adoption of mev-geth, gas fees were artificially high due to MeV-bots "spamming" blocks with higher gas bids to ensure their transaction would occur before the transaction that they were attacking. Since then, it appears as though the majority of the MeV-bots have moved to the flashbots network, relieving the network of this spam that drove gas prices so high. These high gas prices resulted in extra profits for the miners, which I believe is why these two analysis represent MeV as a percentage of transaction fees. Now that the profits that miners enjoy from MeV use an independent channel (flashbots) and mechanism (shared profit vs high gas fees), I'm not sure what would be the best way to represent the additional profits, and whether it is a fair assumption relate it to gas fees. Due to the amount of unknowns that exist around MeV, I think it is fair to use the assumptions made by these two sources as a base representation, which can be improved later once the phenomenon is better understood.